install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -89,6 +89,9 @@ spec:
       annotations:
         summary: Cluster operator has been degraded for 30 minutes.
         description: The {{ "{{ $labels.name }}" }} operator is degraded because {{ "{{ $labels.reason }}" }}, and the components it manages may have reduced quality of service.  Cluster upgrades may not complete. For more information refer to 'oc get -o yaml clusteroperator {{ "{{ $labels.name }}" }}'{{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} or {{ label \"url\" (first $console_url ) }}/settings/cluster/{{ end }}{{ end }}" }}.
+      # The "or..." part of the expr is there for cases Degraded condition is not set at all and based
+      # on how `or + group` by works, it will always be 1 (regardless of `cluster_operator_up` value).
+      # No Degraded condition present -> ClusterOperatorDegraded fires.
```

Comment:
I also tried to explain this in 90539f9a17, although I guess 66fc016e7 makes that harder to find now with `blame`:

```console
$ git blame install/0000_90_cluster-version-operator_02_servicemonitor.yaml | grep -A10 ClusterOperatorDegraded | grep -B2 cluster_operator_up
66fc016e7 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (David Hurta       2022-08-01 13:14:50 +0200 104)             cluster_operator_conditions{job="cluster-version-operator", condition="Degraded"}
66fc016e7 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (David Hurta       2022-08-01 13:14:50 +0200 105)             or on (namespace, name)
66fc016e7 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (David Hurta       2022-08-01 13:14:50 +0200 106)             group by (namespace, name) (cluster_operator_up{job="cluster-version-operator"})
$ git blame 66fc016e7^ install/0000_90_cluster-version-operator_02_servicemonitor.yaml | grep -A10 ClusterOperatorDegraded | grep -B2 cluster_operator_up
90539f9a1 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (W. Trevor King    2021-04-26 12:49:35 -0700  95)           cluster_operator_conditions{job="cluster-version-operator", condition="Degraded"}
d525eb97a install/0000_90_cluster-version-operator_02_servicemonitor.yaml (David Hurta       2022-07-21 12:47:59 +0200  96)           or on (namespace, name)
d525eb97a install/0000_90_cluster-version-operator_02_servicemonitor.yaml (David Hurta       2022-07-21 12:47:59 +0200  97)           group by (namespace, name) (cluster_operator_up{job="cluster-version-operator"})
$ git blame d525eb97a^ install/0000_90_cluster-version-operator_02_servicemonitor.yaml | grep -A10 ClusterOperatorDegraded | grep -B2 cluster_operator_up
90539f9a1 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (W. Trevor King    2021-04-26 12:49:35 -0700  94)           cluster_operator_conditions{job="cluster-version-operator", condition="Degraded"}
90539f9a1 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (W. Trevor King    2021-04-26 12:49:35 -0700  95)           or on (name)
90539f9a1 install/0000_90_cluster-version-operator_02_servicemonitor.yaml (W. Trevor King    2021-04-26 12:49:35 -0700  96)           group by (name) (cluster_operator_up{job="cluster-version-operator"})
```

Maybe we should extend the `description` to cover this situation?